pyanaconda/core/util.py

```diff
@@ -1490,3 +1490,30 @@ def __setattr__(self, name, value):
             return super().__setattr__(name, value)
 
         return setattr(self._object, name, value)
+
+
+_os_version_finder = re.compile(r"VERSION_ID=[\"']?([0-9a-z._-]+)[\"']?")
+
+
+def get_os_version(sysroot=""):
+    """Find version of the OS from the os-release file.
+
+    See os-release(5).
+
+    :param str sysroot: Where to look.
+    :return str: The version
+    """
+    try:
+        with open(sysroot + "/etc/os-release", "r") as f:
+            data = f.read(4096)  # 4 kB should be enough for everyone!
+        return _os_version_finder.findall(data)[0]
+    except (FileNotFoundError, IndexError):
+        pass
+
+    try:
+        with open(sysroot + "/usr/lib/os-release", "r") as f:
```

Comment:
It looks like a duplicate of the code above. You can have a loop that iterates over the file names and returns the first usable result or None.